vanilla-framework/vanilla-framework
# 1682: Make tab selected state sit above bottom border

## Done

- Updated `%pseudo-border--bottom` placeholder to be at `bottom: 0` instead of `bottom: 1px`
- Separated out the common elements of pseudo-border--bottom and pseudo-border--top into a generic placeholder

## QA

- Pull code
- Run `./run serve --watch`
- http://0.0.0.0:8101/vanilla-framework/examples/patterns/tabs/
- Check that the selected and hover state of the tabs list sits above the border, rather than the center

## Details

Fixes #1682
